### PR TITLE
multiarch support for kubernetesui components

### PR DESCRIPTION
**Pull request by myaser**
teapot/issues#3283

### CODE REVIEW

File: cluster/manifests/dashboard/deployment.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ spec:
       serviceAccountName: kubernetes-dashboard
       containers:
       - name: kubernetes-dashboard
-        image: registry.opensource.zalan.do/teapot/kubernetes-dashboard:v2.0.4
+        image: container-registry.zalando.net/teapot/kubernetes-dashboard:v2.0.4-master-13
```
Comment (myaser) [inline cluster/manifests/dashboard/deployment.yaml]:
```suggestion
        image: container-registry.zalando.net/teapot/kubernetes-dashboard:v2.0.4-master-16
```

File: cluster/manifests/dashboard/scraper.yaml [hunk 2]
```diff
@@ -44,7 +44,7 @@ spec:
       serviceAccountName: kubernetes-dashboard
       containers:
       - name: dashboard-metrics-scraper
-        image: registry.opensource.zalan.do/teapot/metrics-scraper:v1.0.4
+        image: container-registry.zalando.net/teapot/metrics-scraper:v1.0.4-master-13
```
Comment (myaser) [inline cluster/manifests/dashboard/scraper.yaml]:
```suggestion
        image: container-registry.zalando.net/teapot/metrics-scraper:v1.0.4-master-16
```

**Comment by myaser** [comment]
👍🏻

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**